orders-service/README.md

```diff
@@ -0,0 +1,150 @@
+
+To see this microservice and Function in action, see [Kyma's getting started guides](link do guida) and learn more about exposing services and Functions through API Rule CRs, bindings their  instances to external application like Redis and subscribes their instances to events from a sample mock application.
+
+## Prerequisites
+
+- Kyma 1.14 or higher. To deploy the Function, [Serverless](https://kyma-project.io/docs/components/serverless/) must be installed on the cluster
+- [Kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) 1.16 or higher
+- [Helm](https://helm.sh/) 3.0 or higher (optional)
+
+## Installation
+
+You can install Orders Service (microservice or Function) either through kubectl or Helm.
+
+### Use kubectl
+
+To install microservice on a Kyma cluster, run:
+
+```bash
+kubectl create ns orders-service
+kubectl apply -f ./deployment/orders-service.yaml
+```
+
+To install the Function on a Kyma cluster, run:
+
+```bash
+kubectl create ns orders-service
+kubectl apply -f ./deployment/function.yaml
+```
+
+### Use Helm
+
+To install microservice on a Kyma cluster, run:
+
+```bash
+helm install orders-service --namespace orders-service --create-namespace --timeout 60s --wait ./chart
+```
+
+See the [`values.yaml`](./chart/values.yaml) file for the configuration of the Helm release.
+
+## Cleanup
+
+See how to remove the example from the cluster through kubectl and Helm.
+
+### Use kubectl
+
+Run this command to completely remove the microservice and all its resources from the cluster:
+
+```bash
+kubectl delete all -l app=orders-service -n orders-service
+kubectl delete ns orders-service
+```
+
+Run this command to completely remove the Function and all its resources from the cluster:
+
+```bash
+kubectl delete all -l app=orders-function -n orders-service
+kubectl delete ns orders-service
+```
+
+### Use Helm
+
+Run this command to completely remove the Helm release with the example and all its resources from the cluster:
+
+```bash
+helm delete orders-service -n orders-service
+kubectl delete ns orders-service
+```
+
+## Configuration
+
+To configure the microservice or the Function, override the default values of these environment variables:
+
+| Environment variable | Description                                                                   | Required   | Default value |
+| ---------------------- | ----------------------------------------------------------------------------- | ------ | ------------- |
+| **APP_PORT**       | Specifies the port of the running service. The function doesn't use this variable. | No | `8080`           |
+| **APP_REDIS_PREFIX**       | Specifies the prefix for all Redis environment variables. See the variables below. | No | `REDIS_`           |
```

Comment:
```suggestion
| **APP_REDIS_PREFIX**       | Specifies the prefix for all Redis environment variables. See the variables in other rows. | No | `REDIS_`           |
```

Reply:
because we rather avoid words like `below` or `above` - my fault, didn't notice that when I was reviewing it for the first time.